client/src/API/ExpenseMethods.ts

```diff
@@ -0,0 +1,39 @@
+import axios from "axios";
+import { IncomeExpenseTransferData, URL } from "../constants";
+
+export const addExpenseItem = async (
+  expenseData: IncomeExpenseTransferData
+) => {
+  try {
+    await axios.post(`${URL}/Expenses/postExpense`, {
+      title: expenseData.title,
+      amount: expenseData.amount,
+      first_name: expenseData.first_name,
```

Comment:
Yeah, but since the backend's DTO has first_name and last_name defined, won't it complain if not given the key/value pair? I wasn't too sure, so I just decided to append for now.